ETL.md

```diff
@@ -0,0 +1,144 @@
+# ETL
```

Comment:
It feels like this is really about cloud diagnostics and less about general ETL. Would this be the beginning of a broader document?

Also, how do users discover this document? Can we link from Settings/Diagnostics/cloud integration docs?